configure.ac

```diff
@@ -5998,6 +6001,48 @@ then
     AM_CFLAGS="$AM_CFLAGS -DHAVE_DH_DEFAULT_PARAMS"
 fi
 
+
+AC_ARG_WITH([max-rsa-bits],
```

Comment:
This also changes DH and DSA, but name is okay with me because you describe it in the comment / help.

Reply:
oh there was a whole thinking process behind it.  explained it to Kaleb thusly:
>oh and @Kaleb J. Himes before you ask, `--with-max-rsa-bits` also sets the DH max bits, which will be noted in the configure help msg, but I needed to name it something that non-cryptographers would grok.  i.e. not `--with-max-classic-asymmetric-bits` :slightly_smiling_face:

Reply:
Hi.